app/src/main/res/layout/item_skill_group_list.xml

```diff
@@ -3,22 +3,22 @@
     android:id="@+id/group_parent"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:background="@color/md_white_1000"
     android:orientation="horizontal"
     android:padding="@dimen/padding_small">
 
     <TextView
         android:id="@+id/group"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="50dp"
```

Comment:
Don't hardcode dimens value

Reply:
Sorry... I created dimen but didn't used it, Squashed all commits and updated according to development.